components/full-width-header/full-width-header.less

```diff
@@ -49,21 +49,36 @@
 	color: @white;
 	
 	@media @screen-medium {
-		padding: @spacing-large @spacing-medium;
+		padding: @spacing-large @spacing-default*1.5;
 	}
 	@media @screen-large {
 		max-width: @column-large-10;
-		flex-direction: row;
 		padding: @spacing-large @spacing-medium;
 	}
+	@media (min-width: 1400px) {
+		flex-direction: row;
+	}
 }
 
 .full-width-header-text-container {
 	display: flex;
 	flex-direction: column;
-	
+	width: 100%;
+	max-width: calc(@column-large-7 - 30);
```

Comment:
Does this work without setting px behind 30?

Reply:
yes it does, dubbel checked. but for clarity I will add px